test/jdk/java/awt/Window/MainKeyWindowTest/TestMainKeyWindow.java

```diff
@@ -383,6 +383,20 @@ public static void main(String[] args)
 
         System.setProperty("apple.laf.useScreenMenuBar", "true");
 
+        if (args.length != 0) {
+            Frame frame = new Frame();
+            MenuBar mb = new MenuBar();
+            mb.add(new Menu("Hello"));
+            frame.setMenuBar(mb);
+            frame.setBounds(400, 180, 1068, 821);
+            frame.setLocationRelativeTo(null);
+            frame.setVisible(true);
+            frame.toFront();
+            Thread.sleep(20_000);
+            System.exit(0);
+            return;
```

Comment:
Can you please explain the reasoning for this snippet where we exit the test. What extra args can be passed? I guess it will exit the whole jtreg vm process cancelling all other tests that are to be run later if we call System.exit?!! or is this only for standalone execution?

Reply:
This part of the code is executed by the test itself see this code above
```
            ProcessBuilder pb = ProcessTools.createJavaProcessBuilder(
                    TestMainKeyWindow.class.getSimpleName(), "mark");
            return ProcessTools.startProcess("Other frame", pb);
```